poh/src/poh_recorder.rs

```diff
@@ -137,17 +137,24 @@ pub struct RecordTransactionsSummary {
     pub starting_transaction_index: Option<usize>,
 }
 
-#[derive(Clone)]
 pub struct TransactionRecorder {
     // shared by all users of PohRecorder
     pub record_sender: Sender<Record>,
     pub is_exited: Arc<AtomicBool>,
 }
 
+impl Clone for TransactionRecorder {
+    fn clone(&self) -> Self {
+        TransactionRecorder::new(self.record_sender.clone(), self.is_exited.clone())
+    }
+}
+
 impl TransactionRecorder {
     pub fn new(record_sender: Sender<Record>, is_exited: Arc<AtomicBool>) -> Self {
         Self {
+            // shared
             record_sender,
+            // shared
             is_exited,
         }
```

Comment:
hmm, seems rebase miss... manual `Clone` is revived unexpectedly...

Reply:
🤦 nice catch. I manually "reverted" my original derive by re-adding these manually.
That's what I get for not using git properly!

Reply:
Added a single commit to make your re-review simpler, which should do nothing once we squash everything on merge: b679dee9c2bc06a1a8a98c1c4af54c5b9d6b9b3f